### PR TITLE
feat(dataset): arg choosing OHLC data

### PR DESCRIPTION
**Pull request by AugustSoderlund**
Updated the `ticker.history()` to retrieve all `["Open", "High", "Low", "Close"]` and an argument in the `to_numpy()` function to select the desired series.

Also updated `fix_missing_data()` to accommodate for the previously mentioned changes.

(And .gitignore to ignore vs code workspace settings)

### CODE REVIEW

**Comment by wilhelmagren** [comment]
Gör en rebase mot main för att få in ändringar från #25 se om de blir några conflicts så löser vi dom

File: finq/datasets/dataset.py [hunk 1]
```diff
@@ -212,38 +222,22 @@ def fix_missing_data(self) -> Dataset:
...
+            )
+            df = (
+                pd.concat(
+                    [
+                        df,
+                        _df_to_append,
+                    ]
                 )
-
-                df = df.sort_index()
```
Comment (wilhelmagren) [inline finq/datasets/dataset.py:246]: Snygg lösning på datumen

**Review by wilhelmagren — COMMENTED** [review]
in function "visualize", np.log(data["Close"]) if log_scale else data["Close"],

should be changed from "Close" to have a kwarg with series with allowed literals, like you did for "as_numpy()"

File: finq/datasets/dataset.py (hunk 1 above)
Comment (AugustSoderlund) [reply finq/datasets/dataset.py:314]: tackar 😄

**Comment by AugustSoderlund** [comment]
Now everything should be fixed 👍

**Comment by AugustSoderlund** [comment]
> Now everything should be fixed 👍

No, the stuff from main didn't come with.
Will fix later. Sry

**Comment by wilhelmagren** [comment]
np :) all incoming changes look good, just need to avoid some conflicts now :P

**Comment by AugustSoderlund** [comment]
@wilhelmagren Now it should contain the latest changes from main! 😄

**Review by wilhelmagren — APPROVED** [review]
Good :+1:

File: finq/datasets/dataset.py [hunk 2]
```diff
@@ -353,12 +346,18 @@ def visualize(
...
+            "Low",
+            "Close",
+        ] = "Close",
     ):
         """ """
 
         for symbol, data in self._data.items():
             plt.plot(
-                np.log(data["Close"]) if log_scale else data["Close"],
+                np.log(data[series]) if log_scale else data[series],
```
[the remark this answers is not in the document]
Comment (wilhelmagren) [reply finq/datasets/dataset.py:362]: See [e2b6c53](https://github.com/wilhelmagren/finq/pull/27/commits/e2b6c53a4b935a066998fa5b168d239298543fbf) and [06d9886](https://github.com/wilhelmagren/finq/pull/27/commits/06d9886e4dad0f953d841ddc3742e2522434c848)

**Pull request merged**